test/selenium_helpers.js

```diff
@@ -34,25 +31,163 @@ const setupSelenium = function(done) {
       })
     }
 
-  this.getLogs = () => {
-    return this.browser.manage().logs().get('browser')
+  browser.getLogs = function(){
+    return this.manage().logs().get('browser')
       .then(logs => console.log('Browser Logs:', logs))
   }
 
-  this.loginAs = loginAs
-  done()
+  browser.loginAs = function(githubId) {
+    return this.visit(`/__login/${githubId}`)
+    .then(session => {
+      const body = this.findElement(By.css('body'), 2000)
+      return this.wait(
+        until.elementTextContains(body, `logged in as ${githubId}`),
+        5000
+      )
+    })
+  }
+
+  browser.closeTabs = function(){
+    return this.getAllWindowHandles()
+      .then(handles => {
+        for(let i = handles.length -1; i > 0; i--){
+          this.switchTo().window(handles[i])
+          this.close()
+        }
+        this.switchTo().window(handles[0])
+      })
+  }
+
+  browser.checkNumberOfTabs = function() {
```

Comment:
this should either be changed to return the number of tabs as an integer or be renamed to `checkIfMoreThanTwoTabs` or some such nameyness